cmd/tiller/tiller.go

```diff
@@ -224,12 +229,13 @@ func start() {
 		}
 	}()
 
-	go func() {
-		mux := newProbesMux()
+	if *probeEnable {
+		go func() {
```

Comment:
There's net add of one `{`, but I don't see where it's closed. The original #8 also indented the `http.ListenAndServe` if block, but that change isn't here.